opam.export

```diff
@@ -247,43 +247,8 @@ installed: [
   "zed.3.1.0"
 ]
 pinned: [
-  "check_opam_switch.~dev"
   "graphql_ppx.1.2.2"
 ]
-package "check_opam_switch" {
```

Comment:
What motivated move of this package from `opam.export` to `scripts/pin-external-packages.sh`?

Reply:
In the nix build, the `~dev` name seems to throw something off severely and opam-nix was unable to proceed.